api_v3/services/LiveStreamService.php

```diff
@@ -724,4 +736,38 @@ public function duplicateTemplateEntry($conversionProfileId, $templateEntryId, $
 		return parent::duplicateTemplateEntry($conversionProfileId, $templateEntryId, $object_to_fill);
 	}
 
+	/**
+	 * Check if entry contains adminTag
+	 *
+	 * @param LiveEntry $entry
+	 * @param string $adminTag
+	 * @return boolean
+	 */
+	private function entryContainsAdminTag(LiveEntry $entry, $adminTag)
```

Comment:
This function better be on entry.php (as the argument is regular entry and not live one in particular)
so signature should be like: public bool isContainsAdminTag(string $adminTag)